roles/restore/defaults/main.yml

```diff
@@ -14,5 +14,15 @@ backup_dir: ''
 # Set no_log settings on certain tasks
 no_log: 'true'
 
+# Default resource requirements
+restore_resource_requirements:
+  limits:
+    cpu: "500m"
+    memory: "512Mi"
+  requests:
+    cpu: "500m"
+    memory: "512Mi"
+
 # Maintain some of the recommended `app.kubernetes.io/*` labels on the resource (self)
 set_self_labels: true
+...
```

Comment:
Is this intentional? This looks like invalid yaml.

Reply:
I take that back, it is valid.  Denotes the end of a yaml file, I just haven't used it yet.  TIL.